app/assets/stylesheets/teams/show.scss

```diff
@@ -26,6 +26,29 @@
     margin: 0;
     padding: 0;
   }
+
+  label[for="team_enrolling"] {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.